scripts/install_sshnpd

```diff
@@ -13,18 +13,26 @@ norm_atsign() {
 
 usage() {
   echo "Usage: $0 [options]"
-  echo "Options:"
-  echo "  -c, --client <address>    (mandatory)  Client address (e.g. @alice_client)"
-  echo "  -d, --device <address>    (mandatory)  Device address (e.g. @alice_device)"
-  echo "  -n, --name <device name>  (mandatory)  Name of the device"
+  echo "General options:"
+  echo "  -u, --update                           Update all services instead of installing"
   echo "  -l, --local <path>                     Install using local zip/tgz"
   echo "  -r, --repo <path>                      Install using local repo"
   echo "  -h, --help                             Display this help message"
+  echo ""
+  echo "Installation options:"
+  echo "  -c, --client <address>    (mandatory)  Client address (e.g. @alice_client)"
+  echo "  -d, --device <address>    (mandatory)  Device address (e.g. @alice_device)"
+  echo "  -n, --name <device name>  (mandatory)  Name of the device"
```

Comment:
Should the mandatory options come first in the usage print?

Reply:
They are only mandatory for installation

Reply:
And technically they aren't mandatory because I ask for them if not provided...